ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -565,9 +558,9 @@ public static void handleInstallDirPermission() {
         try {
             String installationPath = OSUtils.getInstallationPath();
             if (!new File(installationPath).canWrite()) {
-                throw ErrorUtil.createCommandException("current user does not have write permissions.\n\n" +
+                throw ErrorUtil.createCommandException("current user does not have permissions to run the command\n\n" +
                                                                "Current user does not have write permissions to " +
-                                                               "ballerina installation path : " + installationPath +
+                                                               "ballerina installation path " + installationPath +
```

Comment:
Ballerina reference has to be "Ballerina"

Reply:
Already fixed in my next PR.